people.json

```
@@ -5057,16 +5058,28 @@
          "public relations",
          "game audio",
          "marketing",
-         "quality assurance",
+         "music composition",
          "game art"
       ],
       "location": "United States",
       "websites": {
          "personal": [
             [
+			   "Website",
+			   "https://www.retroninja.com/"
+			],
+			[
                "Twitter",
```

This is a bit of a nitpick, but here's my suggestion for spacing for consistency & readability:
```suggestion
		[
			"Website",
			"https://www.retroninja.com/"
		],
		[
			"Twitter",
```